CHANGES/8080.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+Started rejecting ASCII hostnames with invalid characters. For host strings that
+look like authority strings, the exception message includes advice on what to do
+instead.
```

Comment:
I've added bylines to all the present changelog fragments. Let's add one here too, for consistency:
```suggestion
instead -- by :user:`mjpieters`.
```

The preview will be here: https://yarl--954.org.readthedocs.build/en/954/changes.html

Reply:
Sure thing!